src/mpid/ch4/netmod/ofi/ofi_spawn.h

```diff
@@ -433,7 +433,7 @@ static inline int MPIDI_OFI_dynproc_exchange_map(int root,
 
         MPIDI_OFI_PROGRESS_WHILE(!req[1].done || !req[2].done);
         size_t disp = 0;
-        for (i = 0; i < req[0].source; i++) disp += (*remote_upid_size)[i];
+        for (i = 0; (uint64_t) i < req[0].source; i++) disp += (*remote_upid_size)[i];
```

Comment:
@yfguo can we convert the source here to signed? I don't see why we should use a uint type.

Reply:
This PR is still just waiting on a comment from @yfguo. Can you take a look?